server/models.js

```diff
@@ -24,6 +24,27 @@ const User = sequelize.define('user', {
 	location: Sequelize.STRING
 });
 
+const Booking = sequelize.define('booking', {
+	helperId: Sequelize.STRING,
+	clientId: Sequelize.STRING,
```

Comment:
For these ID's, this should be handle through a DB relationship. So a Booking will belong to many Users. From there , through a join table, the ID's will be associated. So we don't need to create properties for them by hand.